devices/YarpRobotLoggerDevice/src/YarpRobotLoggerDevice.cpp

```diff
@@ -115,14 +119,37 @@ YarpRobotLoggerDevice::YarpRobotLoggerDevice()
     // the logging message are streamed using yarp
     BipedalLocomotion::TextLogging::LoggerBuilder::setFactory(
         std::make_shared<BipedalLocomotion::TextLogging::YarpLoggerFactory>());
+
+    m_updatedMetadataSignalVal = 0;
+    sendDataRT = false;
 }
 
 YarpRobotLoggerDevice::~YarpRobotLoggerDevice() = default;
 
 bool YarpRobotLoggerDevice::open(yarp::os::Searchable& config)
 {
+
     constexpr auto logPrefix = "[YarpRobotLoggerDevice::open]";
     auto params = std::make_shared<ParametersHandler::YarpImplementation>(config);
+    std::string remote;
+    sendDataRT = params->getParameter("remote", remote);
+    if (sendDataRT)
+    {
+        yInfo() << "Activating Real Time Logging on yarp port: " << remote;
+        initMetadataFunction = &BipedalLocomotion::YarpRobotLoggerDevice::addChannelAndMetadata;
+        loggingDataFunction = &BipedalLocomotion::YarpRobotLoggerDevice::storeAndSendLoggingData;
+        if (!m_vectorCollectionRTDataServer.initialize(params))
+        {
+            log()->error("Failed to initalize the vectorsCollectionServer", logPrefix);
+            return false;
+        }
+    }
+    else
+    {
+        yInfo() << "Real time logging not activated";
```

Comment:
As before for the `yInfo()`

Reply:
Fixed in [0392a04](https://github.com/ami-iit/bipedal-locomotion-framework/commit/0392a0409591e211c09e5a39379010ff81c7043e)